common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/model/LoginModel.java

```diff
@@ -427,19 +430,23 @@ public boolean userHasPermissionToGetRmThreadDump() {
         return sessionPermissions.get(PERMISSION_RM_GET_RM_THREAD_DUMP);
     }
 
-    public boolean userDoesNotHavePermissionToGetInfrasToPoliciesMapping() {
-        return !sessionPermissions.get(PERMISSION_RM_GET_INFRAS_TO_POLICIES_MAPPING);
+    public boolean userHasPermissionToGetInfrasToPoliciesMapping() {
+        return sessionPermissions.get(PERMISSION_RM_GET_INFRAS_TO_POLICIES_MAPPING);
+    }
+
+    public boolean userHasPermissionToGetSupportedNodeSourceInfras() {
+        return sessionPermissions.get(PERMISSION_RM_GET_SUPPORTED_NODE_SOURCE_INFRASTRUCTURES);
     }
 
-    public boolean userDoesNotHavePermissionToGetSupportedNodeSourceInfras() {
-        return !sessionPermissions.get(PERMISSION_RM_GET_SUPPORTED_NODE_SOURCE_INFRASTRUCTURES);
+    public boolean userDoesNotHavePermissionToDefineNodeSource() {
```

Comment:
For the sake of consistency you should do like upper tests and use `userHasPermissionToDefineNodeSource`